src/plugins/workspace/public/components/workspace_creator/workspace_creator_form.tsx

```diff
@@ -103,7 +103,7 @@ export const WorkspaceCreatorForm = (props: WorkspaceCreatorFormProps) => {
                 <EuiText size="xs">
                   {i18n.translate('workspace.creator.form.associateDataSourceDescription', {
                     defaultMessage:
-                      'Add at least one data source that will be available in the workspace. If a selected data source has related Direct Query data sources, they will also be available in the workspace.',
+                      'Optionally add one or more data sources to make them available in the workspace. If a selected data source has related Direct Query data sources, they will also be available in the workspace.',
```

Comment:
A better version: Optionally select data sources to include in this workspace. Any Direct Query data sources linked to your selections will be automatically included.